jsonscript.go

```diff
@@ -0,0 +1,74 @@
+package templ
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+)
+
+type CSPContextKeyType int
```

Comment:
Remove this in favour of the `templ.GetNonce` function in https://github.com/a-h/templ/pull/752